configuration/framework_config.cfg

```diff
@@ -45,11 +45,11 @@ TARGETS_DIR: targets
 
 
 # ---------------------- Default profile settings ---------------------- #
-DEFAULT_GENERAL_PROFILE: @@@FRAMEWORK_DIR@@@/profiles/general/default.cfg
-DEFAULT_RESOURCES_PROFILE: @@@FRAMEWORK_DIR@@@/profiles/resources/default.cfg
+DEFAULT_GENERAL_PROFILE: ~/.owtf/configuration/general.cfg
+DEFAULT_RESOURCES_PROFILE: ~/.owtf/configuration/resources.cfg
+DEFAULT_MAPPING_PROFILE: ~/.owtf/configuration/mappings.cfg
 DEFAULT_WEB_PLUGIN_ORDER_PROFILE: @@@FRAMEWORK_DIR@@@/profiles/plugin_web/order.cfg
 DEFAULT_NET_PLUGIN_ORDER_PROFILE: @@@FRAMEWORK_DIR@@@/profiles/plugin_net/order.cfg
```

Comment:
@affinity7 I would believe that they are configuration files as well so it would seem logical to move them into the configuration folder. @delta24 any thought?